src/schema.utils.spec.ts

```diff
@@ -81,6 +83,14 @@ describe('schema.utils', () => {
     });
   });
 
+  describe('sanitizeProviderName + desanitizeProviderName', () => {
+    it('returns original provider name', () => {
+      expect(desanitizeProviderName(sanitizeProviderName('foo__bar_baz'))).toBe(
```

Comment:
`foo__bar_baz` is actually the case when sanitizeProviderName + desanitizeProviderName do not return original provider name, but given our options I think this solution is OK

Reply:
😱 you definitely found a bug. I was deciding what to do, but as I was rushing to fix it made this mistake. I will look at fixing it.